QUEST_LV_0200.tsv

```diff
@@ -2800,16 +2800,16 @@ QUEST_LV_0200_20150317_002799	{memo X}There are altars with unknown purposes in
 QUEST_LV_0200_20150317_002800	Merge suddenly appeared when you approaded the altar and started destroying the souls that appeared nearby the altar! Defeat the Merge!
 QUEST_LV_0200_20150317_002801	{memo X}Check the Altar again
 QUEST_LV_0200_20150317_002802	{memo X}Defeated the merge. What were the souls that you saw before the Merge appeared? Check the altar again.
-QUEST_LV_0200_20150317_002803	Defeat the Merge that appeared from the altar
+QUEST_LV_0200_20150317_002803	Defeat Merge
 QUEST_LV_0200_20150317_002804	Cool the altar
-QUEST_LV_0200_20150317_002805	The altar in Oshimas Route keeps fuming red light. Go and check it.
-QUEST_LV_0200_20150317_002806	Spray the cold fluid of monster
+QUEST_LV_0200_20150317_002805	The altar at Oshimas Highway keeps producing a red light. Go and check on it.
+QUEST_LV_0200_20150317_002806	Apply the cold monster fluid.
 QUEST_LV_0200_20150317_002807	{memo X}You can use the cold fluid rarely found in monsters to cool down the wrath of the altar in Oshimas Route. Defeat the monsters in Scattered Stairway and get the cold fluid.
 QUEST_LV_0200_20150317_002808	Spray the monster fluid
-QUEST_LV_0200_20150317_002809	Spray the fluid on Oshimas Altar to cool it down
+QUEST_LV_0200_20150317_002809	Spray the fluid on the Oshimas Highway Altar to cool it down
```

Comment:
Spray the fluid on the Oshimas Highway Altar to cool it down -> Spray the fluid on the altar at Oshimas Highway to cool it down